src/components/Nav.tsx

```diff
@@ -29,6 +29,21 @@ interface SubNavItemProp {
   setNavItemActive: (value: boolean) => void;
 }
 
+const removeDomainFromUrl = (urlString: string): URL | string => {
+  try {
+    const url = new URL(urlString);
+    return `${url.pathname}${url.search}`;
+  } catch (error) {
+    if (error instanceof TypeError) console.error(error.message, urlString);
```

Comment:
Why is special this kind of error?

Reply:
I got an `Object is of type 'unknown'` when I tried to get to message property.

Reply:
Oh, in that case, you should cast with [Error](https://developer.mozilla.org/es/docs/Web/JavaScript/Reference/Global_Objects/Error) in place of TypeError, or am I missing something?

Another option is to simply log the complete error.

Or maybe better, what if you do not catch the error? is it an expected case?